js/widgets/temperament.js

```diff
@@ -1464,9 +1464,11 @@ class TemperamentWidget {
             docById("wheelDiv3").style.zIndex = 10;
             docById("wheelDiv3").style.marginTop = 15 + "px";
             docById("wheelDiv3").style.marginLeft = 37 + "px";
-            docById("wheelDiv3").addEventListener("mouseover", (e) => {
-                this.arbitraryEditSlider(e, angle1, ratios, pitchNumber);
-            });
+            setTimeout(() => {
+                docById("wheelDiv3").addEventListener("mouseover", (e) => {
+                    this.arbitraryEditSlider(e, angle1, ratios, pitchNumber);
+                });
+            },1500);
```

Comment:
How did you decide on this value: 1500ms?

Reply:
Through trial and error.
I tried reducing 50ms from 2000ms and on the 1400ms mark, I felt it could show the "noteInfo" just before the whole animation could complete.
Thus, decided on the value of 1500ms.

Reply:
Should I figure out the time upto 3 decimal places in seconds?